src/Components/Analyzers/src/build/netstandard2.0/Microsoft.AspNetCore.Components.Analyzers.targets

```diff
@@ -0,0 +1,15 @@
+<Project>
```

Comment:
There's spooky magic here, this should definitely have some comments about why we're doing such things.

Reply:
Giant block of comment below.